packages/@glimmer/runtime/lib/compiled/opcodes/component.ts

```diff
@@ -504,7 +504,7 @@ export class ComponentElementOperations implements ElementOperations {
       }
 
       let { element, constructing } = vm.elements();
-      let name = manager.getDebugName(definition.state);
+      let name = manager.getDebugName(state, definition.state);
```

Comment:
Why definition.state here, if the argument is 'definition' (for modifiers)?

Is this an inconsistency between modifiers and components managers?

Reply:
From what i understand: Because state is from the 'main' manager, and definition.state is from the delegate. And state has the delegate needed to delegate to function call